test/Filter.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Filter from "../src/components/Filter";
+
+describe('should renders Filter correctly', () => {
+  const tree = mount(<Filter movieCount={10}/>);
```

Comment:
Try using `shallow` (instead of `mount`) whenever possible - it will help you testing on unit level and is a bit faster.